src/Microsoft.NET.Sdk.Functions/Microsoft.NET.Sdk.Functions.csproj

```diff
@@ -72,7 +72,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="[3.0.23,3.1.0)" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="[3.0.6,3.1.0)" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="[3.0.6,4.1.0)" />
```

Comment:
Do we need any upper bound?

Reply:
Not sure what is best practice. Can remove the upper bound.